user/enterprise/trusty-beta.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Trusty containers for Enterprise (beta)
+layout: en_enterprise
+
+---
+
+**Please note: This is a beta feature. We would definitely like to hear if you run into any problems with the Trusty containers. If you have any questions or run into problems, please write an email to: [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com).**
```

Comment:
Super dig this.

Reply:
Nicee!!  I think we have a special styling for beta features: https://github.com/travis-ci/docs-travis-ci-com/blob/master/STYLE.md#beta-features

Adding both  `BETA` in the beginning and the `{: beta}` at the end :)

Reply:
Also, I'm a big fan of mailto: links with subject lines → maybe we can do hola (mailto:enterprise@travis-ci.com?subject=Trusty%20Beta) instead?